docs/java/features/odata/use-vdm.md

```diff
@@ -20,42 +20,117 @@ import TabItem from '@theme/TabItem';
 
 # Build and execute OData Requests with the Virtual Data Model #
 
-The Virtual Data Model (VDM) allows to build type-safe OData requests for a given service. The java classes represent the _data model and API of the service_. As a consequence all requests that are build through the VDM are not only _syntactically valid_ but also _semantically valid_.
+The Virtual Data Model (VDM) allows to build type-safe OData requests for a given service. The java classes represent the _data model_ and the available _operations_ of the service. As a consequence all requests that are build through the VDM are not only _syntactically valid_ but also _semantically valid_.
 
 ## Using the Fluent API ##
 
-The VDM consistes of _service_ and _data model_ classes. The service classes mirror the API provided by the OData service and serve as entry point for creating requests.
+The VDM consists of _service_ and _data model_ classes. The service classes mirror the API provided by the OData service and serve as entry point for creating requests. They provide a builder which allows for adding further parameters in a fluent way.
 
-To execute HTTP requests the OData client leverages _Destinations_ and are documented in more detail [here](/docs/java/features/connectivity). The following code snippits assume that such a destination is in place:
+To execute HTTP requests the OData client leverages _Destinations_ and are documented in more detail [here](/docs/java/features/connectivity). The following code snippets assume that such a destination is in place:
 
 ```JAVA
```

Comment:
To this and all the following use lower case `java` instead of `JAVA`. You'll see how bright your syntax highlight becomes. All the syntax highlight is lower case. I recommend using: `java`, `py`, `html`, `makefile`, `js`, `jsx`

Reply:
Good catch, thanks for the hint!